fonts.user.styl

```diff
@@ -50,85 +50,148 @@
+:not({ unquote(join(', ', join(', ', icons), join(', ', code), join(', ', composer), 'body > div')) }) {
+    font-family: base       !important if base
+    font-style:  baseStyle  !important if baseStyle
+    letter-spacing: 0px     !important if letterSpacing
+}
+
+:not({ unquote(join(', ', bold)) }) {
+    font-weight: baseWeight            if baseWeight
+}
+
+{ unquote(join(', ', code)) } {
+    font-family: mono       !important if mono
+    font-style:  monoStyle  !important if monoStyle
+    font-weight: monoWeight            if monoWeight
+    letter-spacing: 0px     !important if letterSpacing
+}
+
+{ unquote(join(', ', composer)) } {
     if inputs {
-        // Exclude inputs of button type. #6
-        n += ':not(input:not([type=button])):not(textarea)'
-
-        // Use mono input font on Discord.
-        n += ':not([data-slate-editor=true] *)'
-        n += ':not([class*="channelTextArea-"] *)'
-
-        // Workaround for specificity issues.
-        :not(#x):not(#y):not(#z) {
-            // Discord.
-            [class*="channelTextArea-"],
-
-            // Gmail. #3
-            [contenteditable=true]:not([g_editable]) *,
-            input:not([name="subjectbox"]):not([type=button]),
-
-            textarea {
-                // Feat: set font options if values are not null.
-                font-family: mono       i if mono
-                font-style:  monoStyle  i if monoStyle
-                font-weight: monoWeight i if monoWeight
-            }
-        }
+        font-family: mono       !important if mono
+        font-style:  monoStyle  !important if monoStyle
+        font-weight: monoWeight            if monoWeight
+        letter-spacing: 0px     !important if letterSpacing
+    } else {
+        font-family: base       !important if base
+        font-style:  baseStyle  !important if baseStyle
+        font-weight: baseWeight            if baseWeight
+        letter-spacing: 0px     !important if letterSpacing
+    }
+}
+
+if debug {
+    { unquote(join(', ', join(', ', icons), join(', ', code), join(', ', composer))) } {
+        color: #f00 !important
+    }
+}
+
+@-moz-document regexp('https?://(www\.)?github.com/.*') {
+    { 'body > div' } {
+        font-family: mono       !important if mono
+        font-style:  monoStyle  !important if monoStyle
+        font-weight: monoWeight            if monoWeight
+        letter-spacing: 0px     !important if letterSpacing
     }
 
-    // Feat: set font options if values are not null.
-    { n } {
-        font-family: base       i if base
-        font-style:  baseStyle  i if baseStyle
-        font-weight: baseWeight i if baseWeight
+    if debug {
+        .react-code-lines .react-code-text .react-file-line * {
+            // das was man sieht
+            color: #a0ff !important
+        }
+
+        textarea {
+            // der echte Text
+            color: #0f0 !important
+        }
```

Comment:
This is supposed to get removed before merge, right?

Reply:
I didn't plan to, I hid it behind the new toggle.
I can keep it in my fork only, if you want.